taxonomies/categories.txt

```diff
@@ -27424,19 +28234,25 @@ wikipedia:en:https://en.wikipedia.org/wiki/Quail_eggs
 
 <en:Bird eggs
 en:Duck eggs
-fr:Oeufs de canards
+fr:Oeufs de canards, Oeufs de cane
```

Comment:
I think "oeufs de cane" should be the main name in fact. :)

Reply:
Howard the Duck would beg to differ ;) 
(what about gender equality in duck households ?)
Anyway, it will be fixed in the next commit.